Subject: libxc: fix compilation error with gcc13
From: Charles Arnold carnold@suse.com Wed Jul 6 13:06:40 2022 +0200
Date: Wed Jul 6 13:06:40 2022 +0200:
Git: 8eeae8c2b4efefda8e946461e86cf2ae9c18e5a9

xc_psr.c:161:5: error: conflicting types for 'xc_psr_cmt_get_data'
due to enum/integer mismatch;

Signed-off-by: Charles Arnold <carnold@suse.com>
Reviewed-by: Jan Beulich <jbeulich@suse.com>
Acked-by: Anthony PERARD <anthony.perard@citrix.com>

--- a/tools/include/xenctrl.h
+++ b/tools/include/xenctrl.h
@@ -2516,7 +2516,7 @@ int xc_psr_cmt_get_l3_event_mask(xc_inte
 int xc_psr_cmt_get_l3_cache_size(xc_interface *xch, uint32_t cpu,
                                  uint32_t *l3_cache_size);
 int xc_psr_cmt_get_data(xc_interface *xch, uint32_t rmid, uint32_t cpu,
-                        uint32_t psr_cmt_type, uint64_t *monitor_data,
+                        xc_psr_cmt_type type, uint64_t *monitor_data,
                         uint64_t *tsc);
 int xc_psr_cmt_enabled(xc_interface *xch);
 
